equinix/provider_test.go

```diff
@@ -1,120 +1,11 @@
 package equinix
 
 import (
-	"fmt"
-	"os"
-	"regexp"
-	"strings"
 	"testing"
 
-	"github.com/equinix/ecx-go/v2"
-	"github.com/equinix/terraform-provider-equinix/internal/config"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/stretchr/testify/assert"
 )
 
-var (
```

Comment:
Everything removed here went to `helper_test.go`; while we could put a different file structure in place, most if not all of this code should go away ASAP, so dumping it all in one file is meant as motivation to eliminate this stuff entirely in the future.

Reply:
Another piece of evidence for eliminating the extra `helper_test_test.go` file and just dumping its contents into `helper_test.go`.